addons/web/static/src/js/widgets/data_export.js

```diff
@@ -194,6 +194,28 @@ var DataExport = Dialog.extend({
                 $select.append(new Option(value, export_list_id));
             });
         },
+        'click .o_import_compat': function(){
+            /* When clicking on the Import-Compatible Export hide the User Local Time button*/
+            
+            // Get the export type, along with the timezone option.
+            var $export_type = this.$('.o_import_compat input').filter(':checked').siblings().text();
```

Comment:
Q: Why are the variables being prefixed with `$`?

Reply:
This seemed to be the style that the rest of variables took when returning a DOM.

Reply:
The value of the variable here is just a string right? It's not linked to a jQuery object or anything like that?

Reply:
yep this one needs changing